test/molecule-role/molecule/kubernetes/molecule.yml

```diff
@@ -0,0 +1,43 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: ec2
+lint:
+  name: yamllint
+  options:
+    config-file: .yamllint
+platforms:
+  - name: kubernetes-cluster-agent
+    image: ami-0da760e590e7de0e8    # Ubuntu 18.04 (EBS-Backed x86_64)
+    instance_type: t3.large
+    vpc_subnet_id: subnet-fa36adb2  # eu-west-1a
+    region: eu-west-1
+    groups:
+      - kubernetes_linux_vm
+    ssh_user: ubuntu
+provisioner:
+  name: ansible
+  playbooks:
+    create: create.yml
+    prepare: prepare.yml
+    destroy: destroy.yml
+  lint:
+    name: ansible-lint
+scenario:
+  name: kubernetes
+  test_sequence:
```

Comment:
why do you need to specify the test sequence ?

Reply:
In the initial create of the cluster the default test matrix does a destroy and then cleanup. If there is no cluster, it fails and throws error

Reply:
also for subsequent runs destroy happens before create/prepare. So effectively we are setting capacity to zero twice, which is not required

Reply:
the destroy should run before create because that resources are cleaned up
and what you say is not true: if destroy fails it does NOT throw an error, you can check the other test runs